docs/docs/command-line-interface.mdx

```diff
@@ -419,6 +420,54 @@ This command is most commonly used to import old conversations into Rasa X to an
 them. Read more about [importing conversations into Rasa X](https://rasa.com/docs/rasa-x/installation-and-setup/deploy#1-import-existing-conversations-from-rasa-open-source).
 :::
 
+## rasa evaluate markers
+
+:::caution
+
+This feature is currently experimental and might change or be removed in the future. Share your feedback in the forum to help us make it production-ready.
```

Comment:
Do we have a thread on the forum? It cannot hurt to start a thread right now and point people to it. This way much of the feedback comes in one place.

Reply:
I haven't started a thread, and don't know if someone else has (maybe @TyDunn?). 
Do we have guidelines about this in notion?

Reply:
I have not. Is this something you can work on with @EmmaWightman, @aeshky? I'd happy to review the content before it goes live as well as support you through the process as much as you feel you need :)

Reply:
Sounds good! I'll contact Emma.

Reply:
You're free to contact Emma, but for the Tensorflow 2.6 upgrade, I just took the liberty of starting a thread [on the forum myself](https://forum.rasa.com/t/feedback-upgrading-to-tensorflow-2-6/48331). I can also help you with that if you'd like (Emma's going to be very busy soon with the advent of all the social announcements).

Reply:
Thanks both. How about I start a thread by reusing the content posted [here under "Experimental Feature: Markers"](https://rasa.com/blog/what-s-ahead-in-rasa-open-source-3-0/?utm_campaign=Rasa%20Open%20Source%203.0%20launch&utm_content=187975830&utm_medium=social&utm_source=twitter&hss_channel=tw-2839825685)?

Reply:
[Here](https://docs.google.com/document/d/1OcLvhSsVpGFTSBcHd1YBJFcoZms7lRr5XpzHiSu2o3Q/edit?usp=sharing) is an abridged version.

If you're happy with it, I'll post it on the forum and tag myself and maybe vincent?

Reply:
Text looks good to me 👍  Feel free to start a thread, but I wouldn't share it on Twitter just yet. My impression is that at the moment we're mainly getting feedback from our superhero community because they're guaranteed to be familiar with Rasa. We can share via community slack tho!

Reply:
@aeshky Can we wait to share until after the 3.0 release / live event about markers?

Reply:
@TyDunn Sorry, I already posted it [here](https://forum.rasa.com/t/feedback-markers-a-new-experimental-feature-in-rasa/49344). 😅
I think it might be good if people read a bit about the feature beforehand. We might get better questions that. What do you think? I can remove the post if you disagree :)

Reply:
I still don't think we should post it yet. Can we take it down until the release is out and we have a chance to tell people about it? The timing will be better if we post it right before we have the live event for it

Reply:
Ok done!

Reply:
@TyDunn are you happy for me to post this at the end of the day, prior to the release event on Monday?

Reply:
@aeshky For sure! Now that 3.0 is released it makes sense 🚀